### DISCUSSION TITLE
FEATURE REQ: Download Rate Limiting

### DISCUSSION THREAD

**Discussion by N3EOP** [anidl/multi-downloader-nx#108] [Ideas]
Any chance of a switch to limit the download rate?  Something along the line of wget's --limit-rate=RATE feature would be great.

Currently, it takes all of the download bandwidth, much to the annoyance of He Who Pays The Bills (and needs the Internet for his work).

Thanks.

**Comment by izu-co** [comment]
I don't think there is a way to do that for us, but you can google e.g. "limit application bandwidth" or something like that to find a option for you.
